packages/bpk-component-boilerplate/src/BpkBoilerplate-test.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+import BpkBoilerplate from './BpkBoilerplate';
+
+describe('BpkBoilerplate', () => {
+  it('should render correctly', () => {
+    const tree = renderer.create(<BpkBoilerplate />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should support custom class names', () => {
```

Comment:
We need to remove this and update the snapshots. Otherwise LGTM

Reply:
We want the boilerplate example to support custom class names though, don't we?

Reply:
Sorry I meant the `title` prop

Reply:
Ahh yeah! Now fixed 👍